README.md

```diff
@@ -14,6 +14,6 @@ Include the following in your pom.xml file:
 <dependency>
     <groupId>com.laserfiche</groupId>
     <artifactId>lf-repository-api-client</artifactId>
-    <version>1.0.0</version>
+    <version>1.2.0</version>
```

Comment:
should these changes be merged into the 1.x branch? When we release Java 2.0, we should also update the readme to refer to the correct 2.x version